src/stream/src/executor/aggregation/agg_state.rs

```diff
@@ -60,6 +62,7 @@ fn verify_chunk(ops: Ops<'_>, visibility: Option<&Bitmap>, columns: &[&ArrayImpl
 
 /// State for single aggregation call. It manages the state cache and interact with the
 /// underlying state store if necessary.
+#[derive(EstimateSize)]
```

Comment:
Can you elaborate when we should `derive` and when we have to implement it manually?